docs/core/deploying/single-file.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Single file application
+description: Learn what single file application is and why you should consider using this application deployment model.
+author: lakshanf
+ms.author: lakshanf
+ms.date: 08/28/2020
+---
+# Single file deployment and executable
```

Comment:
This heading says "executable" (singular), and the toc says "executables" (plural). Was the intended?

Reply:
Thanks a lot, Youssef!